tests/test_trigger_out.cpp

```diff
@@ -0,0 +1,7 @@
+#include "sm_tests.h"
+
+/** @brief Fills [SWS_SM_00020] */
+TEST_F(smTests, testInternalStatePropagation) {
+    mySM.triggerOut.SetNotifier(ara::sm::SMStateType::On);
+    EXPECT_EQ(mySM.triggerOut.GetSMState(), ara::sm::SMStateType::On);
```

Comment:
This test shall set SM to ON. Line 5 should be moved to SM

Reply:
Done